packages/google_maps_flutter/lib/src/convert.dart

```diff
@@ -171,7 +171,7 @@ LatLngBounds _convertToBounds(String value) {
           northeast: LatLng(bound['north'] as double, bound['east'] as double));
     }
   } catch (e) {
-    print('Javascript Error: $e');
+    debugPrint('Javascript Error: $e');
```

Comment:
Why don't you fix `kDebugMode` and `print` in line 327?

Reply:
Done?

I'm not sure if it's okay to change the code since I don't know where the code is originally from.

Reply:
https://github.com/flutter/packages/commit/ab463da34bcf8ec80ba1992a8f6e7819f40a360d#diff-ad093591ababe86bff6ff2af9ce718a9b0ef32f011611922b7d245420c71cb27

It looks like a very old remnant...